core/src/sql/v1/value/value.rs

```diff
@@ -2848,16 +2848,20 @@ mod tests {
 
 	#[test]
 	fn check_size() {
-		assert!(64 <= std::mem::size_of::<Value>());
-		assert_eq!(104, std::mem::size_of::<Error>());
-		assert_eq!(104, std::mem::size_of::<Result<Value, Error>>());
+		assert!(
+			64 >= std::mem::size_of::<Value>(),
+			"expected Value to be smaller then 64 bytes found {:?}",
+			std::mem::size_of::<Value>()
+		);
+		assert!(112 >= std::mem::size_of::<Error>());
+		assert!(112 >= std::mem::size_of::<Result<Value, Error>>());
```

Comment:
Also the size seems to change depending on whether the new parser is enabled, which is also weird cause I believe the error type only contains types which are common to both parsers.

Reply:
I think the new sizes are coming from `Value`. Some of the types it's embedding may have new fields or variants.